src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/HiViUap.java

```diff
@@ -58,11 +58,28 @@ public void waitForVideoEnd() {
   }
 
   public void enableVideoToolbar() {
+    if (isMobile()) {
+      enableVideoToolbarMobile();
+    } else {
+      enableVideoToolbarDesktop();
+    }
+  }
+
+  private void enableVideoToolbarDesktop() {
     WebElement videoPlayerElement = wait.forElementVisible(this.videoPlayer);
     Actions builder = new Actions(driver);
     builder.moveToElement(videoPlayerElement).perform();
   }
 
+  private void enableVideoToolbarMobile() {
+    clickVideo();
+  }
+
+  private Boolean isMobile() {
+    final String aClass = driver.findElement(slotSelector).getAttribute("class");
+    return aClass.contains("is-mobile-device") || aClass.contains("theme-mobile-layout");
```

Comment:
Please remove `.theme-mobile-layout`. We're going to remove this class.

Shouldn't we use `is-mobile-layout` instead of `is-mobile-device?`

Reply:
I left two classes to keep backward compatibility. It was implemented while this change wasn't on prod. But thanks, at least there should be a comment or something

Reply:
It's on prod everywhere now